l16/docker/default.env

```diff
@@ -17,3 +17,5 @@ PRYSM_BEACON_VERSION="v0.2.2-dev"
 PRYSM_BOOTSTRAP_NODE=""
 PRYSM_VERBOSITY="info"
 VALIDATOR_DATA_VOLUME="./data/validator_data"
+TEKU_BEACON_VERSION="latest"
```

Comment:
Could you please pin this to whatever is the latest right now?

Reply:
latest release is:`22.6.1`
Would i add this release tag as comment in `default.env`?

Reply:
Yes please, that way we won't have any new validators out there that all of a sudden run 27.0.0 that contains breaking changes :)

Reply:
Thanks for your clarification 👍